redfish-core/lib/service_root.hpp

```diff
@@ -20,12 +20,22 @@
 #include <registries/privilege_registry.hpp>
 #include <utils/systemd_utils.hpp>
 
+#include <filesystem>
+
 namespace redfish
 {
 
 inline void
     handleACFWindowActive(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
 {
+    bool allow;
+    getPropertyAllowUnauthACFUpload(asyncResp, allow); // ignore return code
```

Comment:
Why would you not just return turn/false for if it was allowed?

Reply:
E.g. https://en.cppreference.com/w/cpp/filesystem/is_regular_file

Return value

true if the file indicated by p or if the type indicated by s refers to a regular file, false otherwise. The non-throwing overload returns false if an error occurs. 


If you want the ec out, pass it back..

Reply:
I have rewritten to simplify.